### PR TITLE
Setup the skeleton of the cron job.

### PR DESCRIPTION
**Pull request by dlorenc**
Signed-off-by: dlorenc <dlorenc@google.com>

### CODE REVIEW

**Comment by dlorenc** [comment]
The Start() function here can be called in a goroutine to start the cron server inside the main of the overall server.

**Comment by dlorenc** [comment]
Ref #10

File: pkg/kritis/cron/cron_test.go [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+		fmt.Println("HERE")
+		checked = true
+	}
+	defer func() {
+		podChecker = originalChecker
+	}()
+
+	// Set a deadline of longer than the check interval
+	ctx := context.Background()
+	c, _ := context.WithDeadline(ctx, time.Now().Add(10*checkInterval))
```
Comment (tejal29) [inline pkg/kritis/cron/cron_test.go]: check error.

File: pkg/kritis/cron/cron_test.go [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+	// Speed up the checks for testing.
+	checkInterval = 2 * time.Millisecond
+
+	checked := false
+
+	// Mock the check function and reset after the test.
+	originalChecker := podChecker
+	podChecker = func() {
+		fmt.Println("HERE")
```
Comment (tejal29) [inline pkg/kritis/cron/cron_test.go]: Can remove this.

File: pkg/kritis/cron/cron_test.go (hunk 1 above)
Comment (dlorenc) [reply pkg/kritis/cron/cron_test.go]: It wasn't actually an error, but a manual cancel func. I named it and called it in a defer to be more clear.

File: pkg/kritis/cron/cron_test.go (hunk 2 above)
Comment (dlorenc) [reply pkg/kritis/cron/cron_test.go]: Done.

**Pull request merged**